FoodFriends/app/src/main/res/layout/fragment_profile.xml

```diff
@@ -1,14 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".fragments.ProfileFragment">
 
-    <!-- TODO: Update blank fragment layout -->
+    <ImageView
```

Comment:
I think you're using the constraint layout with the UI tool if I'm not mistaken? I was wondering what was up with the odd dimensions. 

So you'd have to account for different phone sizes as well. Larger phones these images might not look the way you'd like. 

I'd try a Relative or Linear and constrain the boundaries to the width of the screen or other pieces of UI. So it'd be consistent between phone sizes.